README.md

```diff
@@ -1 +1,136 @@
-# chartServer
\ No newline at end of file
+# chartServer:
+
+Server to implement an easy way to generate Chart.
+
+The server will return a PNG image with the chart resulting of the parameter passed on the request.
+
+## Introduction
+
+### Local Installation:
+#### Prerequisites:
+You'll need:
+
+- NodeJS: https://nodejs.org/. This application is tested and supported on v4 LTS.
+  If you use [nvm](https://github.com/creationix/nvm/) to manage your node
+  installations (recommended), you can run `nvm use` in the project directory to
+  select the right version.
+
+  This server need at least the version 6 of node.
+
+#### Installation:
+
+Execute
+
+```
+npm install
+```
+
+## Running
+
+```
+node server
+```
+
+## How it work
+
+This server can generate four types of charts:
+
+- Line
+- Bar
+- Pie
+- Polar area
+- Doughnut
+
+The aspect and values of the chart is passed on the url as parameters.
+All graphics can especify the parameters:
+  - width: Width of the image
+  - height: Height of the image
+  - legend: If you want that the labels of the graphics will be shown and the position.
+
+    legend=[display:[true|false],][position:[top|right|bottom|left]]
+
+The colours are especified as RGB and Aplha in decimal value of three character (if the number is
```

Comment:
Although we normally use hex values for colours, I've used decimal values because I didn't know how to pass a colour with alpha value on HEX to chartjs